build.gradle

```diff
@@ -57,6 +57,8 @@ configurations.all {
         force 'com.google.http-client:google-http-client:1.21.0'
         // the snapshot folder contains a dev version of guava, we don't want to use that.
         force 'com.google.guava:guava:18.0'
+				// force the htsjkd version so we don't get a different one transitively
+				force 'com.github.samtools:htsjdk:2.5.0'
```

Comment:
Can you file a bug against GenomicsDB asking them to fix their htsjdk dependency?